exercises/perfect-numbers/perfect_numbers_test.rb

```diff
@@ -0,0 +1,44 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'example'
+
+class PerfectNumberTest < Minitest::Test
+  def test_aliquot_sum_zero
+    assert_equal 0, PerfectNumber.new(1).aliquot_sum
```

Comment:
Yes, but it's just an idea, I don't think a method called `aliquot_sum` (in the public API) is necessary to solve this problem. :)